src/LiveComponent/src/Resources/doc/index.rst

```diff
@@ -1282,6 +1282,116 @@ and ``live_collection_button_delete`` respectively:
         {{ block('button_widget') }}
     {% endblock live_collection_button_add_widget %}
 
+If you only want to customize some attributes maybe simpler to use the options in the form type:
+
+    // ...
+    ->add('comments', LiveCollectionType::class, [
+        'entry_type' => CommentFormType::class,
+        'allow_add' => true,
+        'allow_delete' => true,
+        'by_reference' => false,
+        'label' => false,
+        'button_delete_options' => [
+            'label' => 'X',
+            'attr' => [
+                'class' => 'btn btn-outline-danger',
+            ],
+        ]
+    ])
+    // ...
+
+If you want more control over how each row is rendered you can override the blocks
+related to the ``LiveCollectionType``. This works the same way as for `the traditional
+collection type`_, but you should use ``live_collection`` and ``live_collection_entry`` as
+prefixes instead.
+
+Let's continue our previous example and customize the rendering of the blog post comments form.
```

Comment:
```suggestion
For example, et's continue our previous example and customize the rendering of the blog post comments form.
```

I'm adding this because, if I'm understand things correctly, the example below is intended to show how you could override the blocks that were mentioned above.